CODEOWNERS

```diff
@@ -1 +1,4 @@
-* @Ebury/online
+# code owners
+* @jaime.perez@ebury.com @milos.cibulka@ebury.com @carlos.matallin@ebury.com @stavros.tomadakis@ebury.com
```

Comment:
Can you remove my email address from this and use my github username instead? I have very bad experience with recruiters scanning github repos for email addresses and then spam irrelevant job offers to that email. I really don't care about working as a PHP developer in Exeter for £15000 a year.

Reply:
Sure don't worry